docs/workflows/api-token.md

```diff
@@ -95,6 +95,22 @@ In the Beyond Identity Admin Console under the "Apps" tab, select the "Beyond Id
 
 ## Revoking Access Tokens
 
+Beyond Identity exposes two different endpoints to revoke a token. Both endpoints achieve the same goal.
+
+There are two differences between them:
+
+- [RFC-7009](https://www.rfc-editor.org/rfc/rfc7009) endpoint needs whole token, management API only needs token_id
+- [RFC-7009](https://www.rfc-editor.org/rfc/rfc7009) endpoint accepts bearer or basic auth, management API accepts only bearer
```

Comment:
Should these both still be RFC-7009 if only one of them is compliant?

And then when you say endpoint needs/accepts ..., management API only needs ... you mean to make it compliant with  RFC-7009 it needs and accepts those but our endpoint only needs or accepts something different?